project-base/storefront/hooks/cart/useIsPaymentWithCartAvailable.ts

```diff
@@ -0,0 +1,9 @@
+import { useSettingsQueryApi } from 'graphql/generated';
+
+export const useIsPaymentWithCartAvailable = (paymentTransactionCount: number) => {
```

Comment:
`canOrderBePaid` or `isPaymentAvailableForOrder` makes more sense to me.

Reply:
But this condition only checks whether the order can be paid by card. if I rename it to canOrderBePaid it won't be relevant to what this hook has inside

Reply:
Then the name does not make sense even more. `useIsPaymentWithCartAvailable` sounds like a hook which tells me if a payment is available for a given CART. The name does not say anything about a CARD.

Reply:
Oh, I see, there's a typo, carT instead of carD, thank you